proto/server.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
```

Comment:
Yo'll have to show me. The Go docs for GRPC don't mention it unless it's buried somewhere... which it may well be, the docs are a little scattered.